src/pages/demo/calendar.tsx

```diff
@@ -1,11 +1,11 @@
-import CalendarPage from "@/components/Calendar";
+import CalendarPage, {CalendarProps} from "@/components/Calendar";
 import { getAuthenticatedUserFromSession } from "@/utils/passage";
 import { getSupabase } from "@/utils/supabase";
-import { AuthProps, ComponentProps } from "@/types/types";
+import { AuthProps } from "@/types/types";
 import { GetServerSideProps } from "next";
 
-export default function calendar({isAuthorized, logOut, data}: ComponentProps) {
-  return (<CalendarPage isAuthorized={isAuthorized} logOut={logOut} data={data} />)
+export default function calendar({isAuthorized, logOut, data, updateEntryDate}: CalendarProps) {
```

Comment:
`logOut` could be a good opportunity to use `useContext` since we are passing this to all the components? just a thought for later

Reply:
This is a great idea! I think I actually may need to go back and change some of this after Calli's pr since we moved the logout button out of nav and changed the function